apps/mealplanr-api/src/collections/category/category.model.ts

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I guess you have removed the eslint comment here. The empty line should also be removed.